react/features/base/lib-jitsi-meet/native/polyfills-browser.js

```diff
@@ -143,24 +143,10 @@ function _visitNode(node, callback) {
             document.cookie = '';
         }
 
-        // Document.querySelector
```

Comment:
Why is this not needed anymore? Do we no longer use the caps plugin?

Reply:
Yes. We don't use it!